model/project_ref.go

```diff
@@ -165,6 +166,13 @@ const (
 	ProjectTriggerLevelBuild = "build"
 )
 
+var adminPermissions = gimlet.Permissions{
```

Comment:
should this go in globals? I'm worried about mutable global data structures.

Reply:
This isn't public. It's just there because the exact struct is used in this package more than once